libusb/os/windows_winusb.c

```diff
@@ -2607,6 +2605,27 @@ static int winusbx_configure_endpoints(int sub_api, struct libusb_device_handle
 	return LIBUSB_SUCCESS;
 }
 
+static HANDLE winusbx_ensure_dev_handle(struct libusb_device_handle *dev_handle, struct winusb_device_priv *priv, struct winusb_device_handle_priv *handle_priv, uint8_t iface)
```

Comment:
Since this is a helper function not exposed anywhere else, I think it shouldn't use the winusbx_ prefix since that seems exclusively used for the API functions. Even if it is only useful for winusbx it would do the right thing for other sub-apis as well. I think "get_device_handle" would be fine.

Reply:
Prefix removed, I like the `ensure` verb as is indicates that something will need to be done if handle is not available already